### ISSUE TITLE
Things to implement (with priority)

### ISSUE THREAD

**Issue by pavel-shliaha** [thermofisherlsms/meth-modifications#4]
Jacob has asked about the priority of implementation of new features in XMLMethodChanger. To explain I need a way to perform tMSn scans with full access to all fragmentation parameters. The implementation does not matter, but I think, given Dereks node selection functionality it is easiest to add the extra fragmentation parameters changes in tMSn and ddMSn. 

 I suggest the following order:

1) I need to understand whether the tMSn methods with UVPD could be created for tune 3.0 in the current version of the workstation I have (runs tune 2.1.1695).  For this please activate UVPD as a fragmentation method and its two settings. If the current version fails I will ask Angie to provide a new workstation (that runs 3.0 tune) 

Then I suggest we finish tMSn settings that are necessary:

2)	Scan description. 
3)	Mz range. 
4)	ScanRange 

After that please add all the parameters that are present in tMSn (including the ones I requested above) to the ddMSn, including:

ActivationType
IsolationWindow
OrbitrapResolution
AgcTarget
Microscans
ETDReactionTime
ETDReagentTarget
ETDSupplementalActivationEnergy
ETDSupplementalActivation
Scan description. 
Mz range. 
ScanRange 

the order does not matter here

**Comment by pavel-shliaha** [comment]
I have had a very productive talk with @dbaileychess and Shannon at ASMS. We discussed that one of our man problems of the fragmentation optimisation experiments is that it is very complicated to deduce which scans belong to which experiments. 

The solution we used in the submitted paper was to add a small mass difference for every condition, e.g. if we target 500 mz, for experiment 1 it will be 500.0001. However, due to some internal instrument bug the mz is not written into the filterString correctly. Derek suggested the easiest way to label scans is to implement Scan Description parameter. I looked at the xsd in the 3.1 brunch, but it does not appear there among "TMSnScan" child nodes. Any news on this front? I can post further description of the mz problem here
